src/screens/HomeScreen.tsx

```diff
@@ -5,15 +5,17 @@ import { SafeAreaView } from 'react-native-safe-area-context';
 import { t } from 'i18next';
 import { HomeStackParamList } from '../navigators/HomeStack';
 import { useActiveAccount } from '../hooks/useActiveAccount';
+import { useAppConfig } from '../hooks/useAppConfig';
 import { ActivityIndicatorView } from '../components/ActivityIndicatorView';
 
 type Props = NativeStackScreenProps<HomeStackParamList, 'Home'>;
 export type HomeScreenNavigation = Props['navigation'];
 
 export const HomeScreen = () => {
   const { isLoading: loadingAccount, account } = useActiveAccount();
+  const { isLoading: loadingAppConfig } = useAppConfig();
 
-  if (loadingAccount) {
+  if (loadingAccount || loadingAppConfig) {
```

Comment:
It might be useful to provide a different activity indicator message depending on the loading state to aid in debugging any issues that might pop up.

Reply:
Will pickup in upcoming cleanup PR